app/views/shared/js/utils.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+var Utils = {
+  importScript: function(src) {
+    var script = document.createElement('script');
+    script.src = src;
+    document.head.appendChild(script);
+
+    return new Promise(function(resolve, reject) {
+      script.addEventListener('load', resolve);
+      script.addEventListener('error', reject);
+    });
+  }
+}
```

Comment:
The goal was to be able to use importScript in the logic iframe as well. Should I redefine this method in every iframe that use it? Note that I only use it to load my Promises.defer method on the logic iframe, so that may be completely overkill. What do you think?